apps/agentstack-server/src/agentstack_server/domain/models/connector.py

```diff
@@ -16,7 +16,8 @@ class AuthorizationCodeFlow(BaseModel):
     authorization_endpoint: AnyUrl
     state: str
     code_verifier: str
-    redirect_url: AnyUrl | None
+    redirect_uri: str
+    client_redirect_uri: AnyUrl | None
```

Comment:
We could do this in non-breaking fashion but given flow is short-lived I opted for renaming to promote clarity.